Tests/Utils/FakeEntityManager.php

```diff
@@ -0,0 +1,536 @@
+<?php
+
+namespace LAG\AdminBundle\Tests\Utils;
+
+use Doctrine\ORM\Configuration;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\NativeQuery;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Doctrine\ORM\PessimisticLockException;
+use Doctrine\ORM\Query;
+use Doctrine\ORM\Query\ResultSetMapping;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\ORM\UnitOfWork;
+
+/**
+ *
+ */
+class FakeEntityManager implements EntityManagerInterface
```

Comment:
It is more convenient to have a fake class when the container builder compile the service in the tests than a mock. I try to compile the builder to test if the all the services are correctly set.